packages/cli/src/databases/migrations/common/1736947513045-CreateTestCaseExecutionTable.ts

```diff
@@ -0,0 +1,47 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+const testCaseExecutionTableName = 'test_case_execution';
+
+export class CreateTestCaseExecutionTable1736947513045 implements ReversibleMigration {
+	async up({ schemaBuilder: { createTable, column } }: MigrationContext) {
+		await createTable(testCaseExecutionTableName)
+			.withColumns(
+				column('id').varchar(36).primary.notNull,
+				column('testRunId').varchar(36).notNull,
+				column('pastExecutionId').int, // Might be null if execution was deleted after the test run
+				column('executionId').int, // Execution of the workflow under test. Might be null if execution was deleted after the test run
+				column('evaluationExecutionId').int, // Execution of the evaluation workflow. Might be null if execution was deleted after the test run, or if the test run was cancelled
```

Comment:
I think these comments could be added as JSDocs comments for these columns into the `TestCaseExecution` class. That way they are a bit more discoverable